### PR TITLE
Iris v2.3.0

### PR DESCRIPTION
**Pull request by pp-mo**
Add Iris v2.3.0 docs

Note also, tweaks to the README.md where this refers to the CLA process + terms.

### CODE REVIEW

**Comment by lbdreyer** [comment]
Built copy available (temporarily) here: https://lbdreyer.github.io/scitools.org.uk/iris/docs/latest/index.html

File: README.md [hunk 1]
```diff
@@ -57,15 +57,24 @@ $ python tools/symlink_common.py .
...
+
+This process will only be complete when all previous contributors have signed the new terms.
+Until then, we respect the rights of all the original contributors, in addition to those on the new list.
+
+Details of the former, original Iris CLA and contributors list were as follows :
+
+    ``contributors.json``, in this repo, contains a JSON dictionary of all contributors who have
+    signed the scitools CLA. It is the definitive source of such contributors.
+    It is where the [CLA bot](https://github.com/SciTools-incubator/scitools-cla-checker)
+    looks to determine if a CLA is required.
```
Comment (lbdreyer) [inline README.md]: The CLA bot must also be looking at the new list:
https://script.google.com/macros/s/AKfycbwqZ0JLlOKJEcvYUYFXo77o6nux9tSHvOJISUnk3yHi-Bk0Qanz/exec.

Comment (pp-mo) [reply README.md]: Yes, I see it also [reads the old list](https://github.com/SciTools-incubator/scitools-cla-checker/blob/master/scitools_cla_checker/update_pr.py#L202), so for now it combines both.

I will add something to make this clearer.

**Comment by pp-mo** [comment]
Thanks @lbdreyer 
Does that fix it for you ?

**Pull request merged**